PocketKit/Sources/Analytics/AppEvents/Home.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  File.swift
+//  
+//
+//  Created by Daniel Brooks on 2/17/23.
+//
+
+import Foundation
+public extension Events {
+    struct Home {}
+}
+
+public extension Events.Home {
+    /**
+     Fired when a card in the `Recent Saves` section scrolls into view
```

Comment:
@JohnOberhauser I think it's prob a good idea to add the descriptions as doc blocks

Reply:
I will start doing this as well for Search in future PRs